server/src/db.js

```diff
@@ -115,9 +124,11 @@ exports.transaction = function(func) {
 exports.del = exports.update;
 
 exports.exec = function(sql, args) {
+  let timer = initTiming();
   return getConnection().then(function([client, done]) {
```

Comment:
Would it be useful to track slow connection times as well? Maybe shove a second timer in here?